react/Section.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { FC } from 'react'
+import { useCssHandles, applyModifiers } from 'vtex.css-handles'
+import { useResponsiveValue } from 'vtex.responsive-values'
+
+interface Props {
+  borderless?: boolean
+  marginBottom?: number
+  paddingBottom?: number
```

Comment:
Couldn't these props be `MaybeResponsiveInput`? And maybe `borderless` too.

Reply:
Can do!